cypress/integration/examples/cypress_api.spec.js

```diff
@@ -0,0 +1,219 @@
+/// <reference types="cypress" />
+
+context('Cypress.Commands', () => {
```

Comment:
Some of the tests are checking Cypress default functions and properties. Remove unnecessary tests and focus on the task requirements.